<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Qt App</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QTableView" name="rentTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>260</width>
      <height>380</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="showUsers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Users</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showBooks">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Books</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showRentals">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rented books</string>
    </property>
   </widget>
   <widget class="QListView" name="userList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>260</width>
      <height>380</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="bookList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>260</width>
      <height>380</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bookEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bookEdit2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="rentUserCombo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="rentBookCombo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rentButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="returnUserCombo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="returnBookCombo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="manageButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>manage</string>
    </property>
   </widget>
   <widget class="QLabel" name="rentalText1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>add rental</string>
    </property>
   </widget>
   <widget class="QLabel" name="rentalText2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>remove rental</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
